src/ui/public/agg_types/controls/number_interval.html

```diff
@@ -6,15 +6,6 @@
       position="'right'"
       content="'Interval will be automatically scaled in the event that the provided value creates more buckets than specified by Advanced Setting\'s histogram:maxBars'"
     ></icon-tip>
-
-    <icon-tip
```

Comment:
why are we changing the `number_interval` template ? its used by `histogram` not `date_histogram` ?

Reply:
@ppisljar I changed the previous `warning` config to `help` instead to reflect the purpose better (to help users understand limitations). `histogram` supported `warning` so its template has been updated to use `help` here and will display the text below the input field.